activerecord/lib/active_record/locking/optimistic.rb

```diff
@@ -150,7 +150,7 @@ def locking_column=(value)
 
         # The version column used for optimistic locking. Defaults to +lock_version+.
         def locking_column
-          reset_locking_column unless defined?(@locking_column)
+          @locking_column = DEFAULT_LOCKING_COLUMN unless defined?(@locking_column)
```

Comment:
@rafaelfranca looked through history of this file (didn't branch out) and even wondering if `locking_column=` or `reset_locking_column` even needs to reload the schema.